codegen/ui_tests/rhai_fn_rename_collision_oneattr_multiple.stderr

```diff
@@ -0,0 +1,17 @@
+error: duplicate Rhai signature for 'get$bar'
+  --> $DIR/rhai_fn_rename_collision_oneattr_multiple.rs:17:15
+   |
+17 |     #[rhai_fn(get = "bar")]
+   |               ^^^
+
+error: duplicated function renamed 'get$bar'
+  --> $DIR/rhai_fn_rename_collision_oneattr_multiple.rs:12:15
+   |
+12 |     #[rhai_fn(name = "foo", get = "bar")]
+   |               ^^^^
```

Comment:
Can you make it point to the `get` instead of the `name`?

If not, then write the test for a `name` collision instead of a `get` collision (as I wrote in my original comment). That should be easier, and I expect to change error handling in properties as part of my getter-setter work.

Reply:
How can I make it point to `get`?  I tested for a `get` collision just to make sure that it works...

Reply:
The CI on GitHub actually points to the entire attribute parameter length, not individual element.